scripts/pegas.js

```diff
@@ -34,7 +34,8 @@ module.exports = function(robot) {
 
       $('.job-list .job').each(function() {
         var title = $(this).find('h4').text().replace(/\n|\r/g, ' ');
-        var link = 'https://www.getonbrd.cl' + $(this).find('a').attr('href');
+        var path = $(this).find('a').attr('href');
+        var link = path.indexOf('https://') === -1 ? 'https://www.getonbrd.cl' + $(this).find('a').attr('href') : path;
```

Comment:
Con esto simplificas la concatenacion y se da soporte a urls con `http://`
```suggestion
        var link = /https?:\/\//.test(path) ? `https://www.getonbrd.cl${path}` : path;
```

Reply:
apoyo eso y c mergea